db/models/order.js

```diff
@@ -0,0 +1,16 @@
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Order = db.define('orders', {
+  orderDate: {
+    // is this necessary if we have a created_at we can use??
+    type: Sequelize.DATE,
+  },
+  status:{
+    // only admin should be able to change the status
+    type: Sequelize.ENUM,
+    values: ['processing', 'shipped', 'delivered']
```

Comment:
sorry for so many comments @cleverettgirl - also we need an shipping address, and we need to add associations between user and order and once orderlines are done, add that association too?